html-css/week1/index.html

```diff
@@ -4,30 +4,37 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>My Portfolio</title>
+    <title>Bogdan Dobinda Front-end Developer½</title>
     <link rel="stylesheet" href="style.css" />
     <link rel="stylesheet" href="mediaqueries.css" />
   </head>
   <body>
     <nav id="desktop-nav">
-      <div class="logo"><img src="./assets/logo3.png" alt="PrivatLogo" /></div>
-      <div>
+      <img 
+      class="logo" 
+      src="./assets/logo3.png" 
+      alt="PrivatLogo" />
+
+      <nav>
         <ul class="nav-links">
           <li><a href="#about">About</a></li>
           <li><a href="#experience">Experience</a></li>
           <li><a href="#projects">Projects</a></li>
           <li><a href="#contact">Contact</a></li>
         </ul>
-      </div>
+      </nav>
     </nav>
     <nav id="hamburger-nav">
-      <div class="logo">John Doe</div>
+      <img 
+      class="logo" 
+      src="./assets/logo3.png" 
+      alt="Logo for Creative Design by Bogdy Alex" />
       <div class="hamburger-menu">
-        <div class="hamburger-icon" onclick="toggleMenu()">
+        <button class="hamburger-icon" onclick="toggleMenu()" role="button">
```

Comment:
<button> tags don't need `role="button"` as that's their default role. The attribute `role` was a suggestion if you would have continued using <div> but you wanted it to work as a <button>

Reply:
I understand now about this role button. I can use it only if I am using div as a button for some reasons, otherwise if I use button tag then I don't need to use , as is by default.
I couldn't see your comment complete but I assume that this is the answer.
Thanks.